editor/components/EMSLinkRecordButton.tsx

```diff
@@ -14,37 +16,58 @@ export interface EMSLinkRecordButtonProps extends Record<string, unknown> {
 
 const EMSLinkRecordButton: React.FC<
   RowActionComponentProps<EMSPatientCareRecord, EMSLinkRecordButtonProps>
-> = ({ record, additionalProps }) => {
-
+> = ({ record, additionalProps, mutation, onSaveCallback }) => {
   const isLinkingInProgress = !!additionalProps?.selectedEmsPcr;
   const isLinkingThisRecord =
     additionalProps?.selectedEmsPcr &&
     record.id === additionalProps?.selectedEmsPcr.id;
 
-  const isRecordAlreadyLinked = !!record.person_id;
-
-  if (isRecordAlreadyLinked) {
-    return null;
```

Comment:
i made the executive decision to keep the buttons around even if the record has been matched, incase you clicked the wrong person and want to match it to another without having to manually type in the person id